readthedocs/projects/admin.py

```diff
@@ -177,6 +186,61 @@ def delete_selected_and_artifacts(self, request, queryset):
                 broadcast(type='app', task=remove_dir, args=[project.doc_path])
         return delete_selected(self, request, queryset)
 
+    def mark_as_abandoned(self, request, queryset):
+        """
+        Marks selected projects as abandoned.
+
+        It adds '-abandoned' to the slug, marks
+        Project.is_abandoned and sends an email to the user
+        notifying the change and link to the new project/docs
+        page.
+        """
+        # Change project's slug and mark it as abandoned.
+        queryset.update(
+            slug=Concat('slug', Value('-abandoned')),
```

Comment:
I think if we change the slug without renaming the directories where we the HTML are hosted in the webs, we will end up with broken docs. Like, `myproject-abandoned.readthedocs.io` won't work because there is no directory called `myproject-abandoned` on disk.

I suppose that we will need to broadcast a task to the webs for this.